README.rst

```diff
@@ -444,6 +453,25 @@ if needed you can assign a callback to be ran after a query or set of queries co
     def _handle_insert_success(item_dict):
         #  callback logic here happens after the transaction is complete
 
+In order to do this, set use_get_last_insert_id=True in the sqludate decorator, and add a get_last_insert_id method as a kwarg to your
+function. This method will be overwritten by the sqlupdate decorator so just set it equal to None.
```

Comment:
`This method is a placeholder kwarg that will be automatically overwritten by the sqlupdate decorator at run time. Therefore, the assigned value in the function definition does not matter.`